zookeeper-api/src/test/java/org/apache/helix/zookeeper/impl/client/TestRawZkClient.java

```diff
@@ -89,29 +89,6 @@ public void afterClass() {
     _zkClient.close();
   }
 
-  @Test
-  void testUnimplementedTypes() {
-    // Make sure extended types are disabled
-    System.clearProperty("zookeeper.extendedTypesEnabled");
```

Comment:
Why is this test removed? @rabashizade

Reply:
It relies on the `zookeeper.extendedTypesEnabled` not being set. Unsetting this property runs the risk of failing parallel tests that rely on it being set. Since this test just checks if ZK server throws the expected type of exception, I thought it's okay to remove it (I had added it myself in [this PR](https://github.com/apache/helix/pull/2090/files#diff-709c5951f5f7f2abe19a5641bdd677e3413bc46304e32eb29ba3513cc48029e1R93) for this feature) given this is really the ZK server behavior, and keeping it makes other tests unstable.